docroot/modules/custom/mass_views/mass_views.module

```diff
@@ -229,3 +229,27 @@ function mass_views_do_invalidate($entity): void {
     Cache::invalidateTags($tags);
   }
 }
+
+/**
+ * Implements hook_views_pre_render().
+ */
```

Comment:
Unclear to me what this function is doing. Please add code comment.

Reply:
Added some more info to describe the usage.